pages/badges/badge-single.jsx

```diff
@@ -16,16 +16,19 @@ var React = require('react'),
     Modal = require('../../components/modal.jsx');
 
 var Navigation = React.createClass({
+  contextTypes: {
+    intl: React.PropTypes.object
+  },
   render: function() {
     var prev = this.props.prev;
     var next = this.props.next;
     return (
       <div className="badge-navigation">
-        <a className="previous" href={prev.url}>
+        <a className="previous" href={"/" + this.context.intl.locale + prev.url}>
           <img src={prev.img} />
           <span clasName="label">← {prev.title}</span>
         </a>
-        <a className="next" href={next.url}>
+        <a className="next" href={"/" + this.context.intl.locale + next.url}>
```

Comment:
it makes it very easy to check files for use of raw `<a>` though, which is easily automated.